### ISSUE TITLE
Video player controls don't work and loading never disappears

### ISSUE THREAD

**Issue by i-lost-my-bagel** [Rehike/Rehike#451]
### Rehike version

0.8.0

### Operating system

Linux 6.1.0-15-amd64 #1 SMP PREEMPT_DYNAMIC Debian 6.1.66-1 (2023-12-09) x86_64 

### PHP version

8.2.7

### What's going on?

https://github.com/Rehike/Rehike/assets/32345477/96b9f25a-05fc-4c9e-bf70-5c825d6c6ca6

Clicking around on the video player options does nothing and the loading signs never go away.

**Comment by i-lost-my-bagel** [comment]
https://github.com/Rehike/Rehike/assets/32345477/e66da21d-c52e-4a86-93ab-b58e81f140af

Also when going to another video it started playing the previous one again and now the video player controls just don't show up

**Comment by RealPortalPlayer** [comment]
YouTube did something, because I can confirm that it's happening to me too

**Comment by RealPortalPlayer** [comment]
Here's the error that gets spammed in console

> Uncaught TypeError: this.j[(h + 1)].apply is not a function
    oa https://www.youtube.com/s/player/0d02ada0/player_ias.vflset/en_US/base.js:8379
    oa https://www.youtube.com/s/player/0d02ada0/player_ias.vflset/en_US/base.js:10388
    qo https://www.youtube.com/s/player/0d02ada0/player_ias.vflset/en_US/base.js:10383
    vc https://www.youtube.com/s/player/0d02ada0/player_ias.vflset/en_US/base.js:10385
    onVideoProgress https://www.youtube.com/s/player/0d02ada0/player_ias.vflset/en_US/base.js:12600
    oa https://www.youtube.com/s/player/0d02ada0/player_ias.vflset/en_US/base.js:8379
    oa https://www.youtube.com/s/player/0d02ada0/player_ias.vflset/en_US/base.js:8761
    wQ https://www.youtube.com/s/player/0d02ada0/player_ias.vflset/en_US/base.js:12448
    Dd https://www.youtube.com/s/player/0d02ada0/player_ias.vflset/en_US/base.js:305
    dispatchEvent https://www.youtube.com/s/player/0d02ada0/player_ias.vflset/en_US/base.js:7793
    dispatchEvent https://www.youtube.com/s/player/0d02ada0/player_ias.vflset/en_US/base.js:9583
    listener https://www.youtube.com/s/player/0d02ada0/player_ias.vflset/en_US/base.js:5063

**Comment by ephemeralViolette** [comment]
Temporary fix:

```json
{
    "expire": 9999999999999,
    "content": {
        "baseJsUrl": "\/s\/player\/dee96cfa\/player_ias.vflset\/en_US\/base.js",
        "baseCssUrl": "\/s\/player\/dee96cfa\/www-player.css",
        "embedJsUrl": "\/s\/player\/dee96cfa\/www-embed-player.vflset\/www-embed-player.js",
        "signatureTimestamp": "19697"
    }
}
```

**Comment by YukisCoffee** [comment]
Fixed in dc59150c24cc66f340a8ccbcdfd7dd682f8035bd.

They ended up removing a legacy function for event handler resolution from the player, which broke how Hitchhiker's JS interfaces with the player. To correct this, I just made Hitchhiker JS use the modern method by modifying the Hitchhiker base JS itself.
